ingestor/src/main/scala/uk/ac/wellcome/ingestor/controllers/ManagementController.scala

```diff
@@ -0,0 +1,16 @@
+package uk.ac.wellcome.platform.ingestor.controllers
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import com.twitter.finagle.http.Request
+import com.twitter.finatra.http.Controller
+
+
+@Singleton
+class ManagementController @Inject()() extends Controller {
+  get("/management/healthcheck") { request: Request =>
+    response.ok.json(Map("message" -> "ok"))
```

Comment:
Maybe we should include the app name here? We now have several healthcheck endpoints that just return

```json
{"message": "ok"}
```

It might be nice to be able to distinguish by output alone.